tools/osv-scanner/cmd/main.go

```diff
@@ -11,8 +11,8 @@ import (
 	"path/filepath"
```

Comment:
Rename this to `tools/osv-scanner/main.go` ?

Reply:
Isn't this https://pkg.go.dev/path/filepath ?

Reply:
I meant the file itself.

Reply:
Oh, got it, move it out of the cmd directory.

Reply:
Addressed in 49ea940